### PR TITLE
Sort imports

### PR DESCRIPTION
**Pull request by ludeeus**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/entrypoints/authorize.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
 // Compat needs to be first import
```
Comment (bramkragten) [inline src/entrypoints/authorize.ts:1]: ^

File: src/entrypoints/core.ts [hunk 2]
```diff
@@ -1,6 +1,4 @@
 // Compat needs to be first import
```
Comment (bramkragten) [inline src/entrypoints/core.ts:1]: ^

File: src/entrypoints/custom-panel.ts [hunk 3]
```diff
@@ -1,10 +1,10 @@
-import "../resources/compatibility";
```
Comment (bramkragten) [inline src/entrypoints/custom-panel.ts:1]: Should be the first

File: src/entrypoints/onboarding.ts [hunk 4]
```diff
@@ -1,9 +1,9 @@
 // Compat needs to be first import
```
Comment (bramkragten) [inline src/entrypoints/onboarding.ts:1]: ^

File: demo/src/ha-demo.ts [hunk 5]
```diff
@@ -1,11 +1,11 @@
-import "../../src/resources/compatibility";
```
Comment (bramkragten) [inline demo/src/ha-demo.ts:1]: Should be first

File: src/entrypoints/custom-panel.ts (hunk 3 above)
Comment (ludeeus) [reply src/entrypoints/custom-panel.ts:1]: That's just annoying

File: .devcontainer/devcontainer.json [hunk 6]
```diff
@@ -24,7 +24,16 @@
     "editor.formatOnSave": true,
     "editor.formatOnType": true,
     "[typescript]": {
-      "editor.defaultFormatter": "esbenp.prettier-vscode"
+      "editor.defaultFormatter": "esbenp.prettier-vscode",
+      "editor.codeActionsOnSave": {
+        "source.organizeImports": true
```
Comment (bramkragten) [inline .devcontainer/devcontainer.json:29]: Personally, I've had very bad experiences with this, super slow and messed up the code more than once when I saved it again while it was still processing.

Comment (ludeeus) [reply .devcontainer/devcontainer.json:29]: because of `/compatibility` I have to disable it anyway...

**Review by bramkragten — APPROVED** [review]

**Pull request merged**